.github/workflows/auto-assign-reviewer.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Auto Assign'
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  add-reviews:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kentaro-m/auto-assign-action@v1.2.0
```

Comment:
I'm not 100% sure this is causing the action failure, but the docs for this action mention that it expects the auto assign file to be at `.github/auto-assign.yml` and it's currently located `.github/workflows/auto-assign.yml`. If you want to keep it where it is, then it looks like you can just add a config field here to tell it where the file is. I think it would be better to just move the `auto-assign.yml` out of the `workflows` folder, since it's not actually a workflow.

Reply:
On it!

Reply:
@aswanson-nr: I'm not entirely sure how to check if the action succeeded, but it looks like all the checks passed!